site/class-2-debug/Nina92/readme.md

```diff
@@ -0,0 +1,43 @@
+# Debugging Code
+
+## Description
+I debugged the code from [`site/class-2/debug`](https://github.com/cmda-fe3/course-17-18/blob/master/site/class-2/debug).
+
+## Background
+This are the bugs I fixed:
+
+### HTML
+* Added `<head>` element
+* Added `<body>` element
+* Changed `href=../index.css` from `<link>` to `href=index.css`
+* Changed `src=../index.js` from `<script>` to `src=index.js`
+* Changed `src=https://d3js.org/d3.v3.min.js` from `<script>` to `src=https://d3js.org/d3.v4.min.js`
+
+### CSS
+* Changed `width: 100w` to `width: 100vw` 
+* Changed `height: 100h` to `height: 100vh` 
+* Changed `fill: no` to `fill: none`
+* Changed `stroke: currentcolor` to `stroke: currentColor`
```

Comment:
Both `currentColor` and `currentcolor` are fine btw, not a bug :)

Reply:
Ah ok, good to know! I had never heard of it so I googled it and the first link said that it was written in camelCase so I assumed it was a bug.